src/Credentials.js

```diff
@@ -168,11 +169,12 @@ class Credentials {
     if (params.callbackUrl) {
       payload.callback = params.callbackUrl
     }
-    if (params.network_id) {
-      payload.net = params.network_id
+    if (params.networkId) {
+      payload.net = params.networkId
     }
+
     if (params.accountType) {
-      if (['general', 'segregated', 'keypair', 'devicekey', 'none'].indexOf(params.accountType) >= 0) {
+      if (['general', 'segregated', 'keypair', 'none'].indexOf(params.accountType) >= 0) {
```

Comment:
very minor and pedantic point, but would it be clearer to use `Array.prototype.includes` here instead since the actual index value is not being used anyway except for calculating a boolean anyway?

Reply:
I'm with ya on that, maybe a future PR can clean up some of the less clear/idiomatic bits such as this